lib/json_rails_logger/version.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 module JsonRailsLogger
+  NAME = "json_rails_logger"
+  OWNER = "epimorphics"
```

Comment:
We should change the `authors` of the gemspec to be Epimorphics (.gemspec allows the singular `author` as well as `authors`), and then we can read the value from the gemspec in the same way as the `name`, above

Reply:
The case is significant. It's the location of the github registry url , it's not just boiler plate. So it needs to be `epimorphics` not "Epimorphics Ltd". Are you suggesting having both?
```
s.author = 'epimorphics'
s.authors = 'Epimorphics Ltd'
```

Reply:
OK, so:

```
# gemspec
s.author = 'Epimorphics'

# Makefile
OWNER?=$(shell ruby -e "puts Gem::Specification::load('json_rails_logger.gemspec').author.downcase")
```